echo.go

```diff
@@ -586,6 +603,20 @@ func (e *Echo) Stop() error {
 	return e.server.Stop()
 }
 
+// Add meta information about endpoint using MiddlewareFunc
+func (e *Echo) MetaMiddleware(d H, handler MiddlewareFunc) MiddlewareFunc {
+	name := handlerName(handler)
```

Comment:
Value generated same way as for anonymous handler function. Value is something like github.com/pavelsmejkal/echo.TestEchoMeta.func1 in my case. func(NUMBER) dependents on the order of definition. 

middleware github.com/pavelsmejkal/echo.TestEchoMeta.func1
handler       github.com/pavelsmejkal/echo.TestEchoMeta.func2